Assets/Scripts/Interaction Scripts/ThirdPersonView.cs

```diff
@@ -0,0 +1,450 @@
+﻿namespace ISAACS {
+
+    using System.Collections;
+    using System.Collections.Generic;
+    using UnityEngine;
+        using VRTK;
+        using ISAACS_ControllerInput;
+
+
+    /* TODO
+    */
+
+    public class ThirdPersonView : MonoBehaviour
+    {
+        public GameObject LeftController;
+        public GameObject LeftUI;
+        public GameObject RightController;
+        public GameObject RightUI;
+
+        private VRTK_Pointer LeftPointer;
+        private VRTK_BezierPointerRenderer LeftPointerRenderer;
+        private VRTK_Pointer RightPointer;
+        private VRTK_StraightPointerRenderer RightPointerRenderer;
+        private VRTK_UIPointer RightUIPointer;
+
+        public ControllerInput controllerInput; /// Initiate a singleton of the ControllerState class. Assign it to the Controller GameObject.
+
+        private enum ControllerState { IDLE, SCALING, SELECTING_DRONE, SELECTING_SENSOR, AIMING_SELECTOR, SETTING_WAYPOINT_HEIGHT, PLACING_WAYPOINT, MOVING_WAYPOINT, UNDOING, REDOING }
+        private ControllerState controllerState;
+
+        public GameObject Pivot; /// pivot is the center of the table
```

Comment:
what is the point of the pivot when you are rotating world already?

Reply:
pivot is the object around which we're rotating